src/input-nested-select/input-nested-select.tsx

```diff
@@ -156,10 +156,11 @@ export const InputNestedSelect = <V1, V2, V3>({
     // HELPER FUNCTION
     // =========================================================================
     const getDisplayValue = (): string => {
```

Comment:
generally:
1. custom stringify takes precedence
2. else stringify value
3. fallback to `""` if value is undefined/null